src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -170,12 +170,19 @@
          (get-in ws ["workspace" "bucketName"])])
       (style/create-section-header "Analysis Submissions")
       (style/create-paragraph
-        (let [fail-count (->> submissions
-                           (filter (complement all-success?))
-                           count)]
-          (str (count submissions) " Submissions"
-            (when (pos? fail-count)
-              (str " (" fail-count " failed)")))))]]))
+        (let [success-count (->> submissions (filter all-success?) count)
+              running-count (->> submissions (filter any-running?) count)
+              fail-count (- (- (count submissions) success-count) running-count)]
```

Comment:
`-` can take multiple arguments.  So `(- (count submissions) success-count running-count)` should work.